docs/dev/polyfill.md

```diff
@@ -0,0 +1,34 @@
+- Add it to entry points of those directories where it's required:
+  - [`packages/core-js/es`](https://github.com/zloirock/core-js/blob/master/packages/core-js/es)
+  - [`packages/core-js/stable`](https://github.com/zloirock/core-js/blob/master/packages/core-js/stable)
+  - [`packages/core-js/actual`](https://github.com/zloirock/core-js/blob/master/packages/core-js/actual)
+  - [`packages/core-js/full`](https://github.com/zloirock/core-js/blob/master/packages/core-js/full)
+  - [`packages/core-js/proposals`](https://github.com/zloirock/core-js/blob/master/packages/core-js/proposals)
+  - [`packages/core-js/stage`](https://github.com/zloirock/core-js/blob/master/packages/core-js/stage)
+  - [`packages/core-js/web`](https://github.com/zloirock/core-js/blob/master/packages/core-js/web).
+- Add unit tests to [`tests/unit-global`](https://github.com/zloirock/core-js/blob/master/tests/unit-global) and [`tests/unit-pure`](https://github.com/zloirock/core-js/blob/master/tests/unit-pure).
+- Add tests of entry points to [`tests/entries/unit.mjs`](https://github.com/zloirock/core-js/blob/master/tests/entries/unit.mjs).
+- Make sure that you are following [our coding style](#style-and-standards).
+- Document it in [README.md](https://github.com/zloirock/core-js/blob/master/README.md), [CHANGELOG.md](https://github.com/zloirock/core-js/blob/master/CHANGELOG.md) and [docs](./docs.md#add-doc-for-polyfill)
+
+## Style and standards
+
+The coding style should follow our [ESlint configuration](https://github.com/zloirock/core-js/blob/master/tests/eslint/eslint.config.js). You can test it by calling [`npm run lint`](./testing.md). Different places have different syntax and standard library limitations:
+
+- Polyfill implementations should use only ES3 syntax and standard library, they should not use other polyfills from the global scope.
+- Unit tests should use the modern syntax with our [minimalistic Babel config](https://github.com/zloirock/core-js/blob/master/babel.config.js). Unit tests for the pure version should not use any modern standard library features.
+- Tools, scripts and tests, performed in NodeJS, should use only the syntax and the standard library available in NodeJS 8.
+
+File names should be in the kebab-case. Name of polyfill modules should follow the naming convention `namespace.subnamespace-where-required.feature-name`, for example, `esnext.set.intersection`. The top-level namespace should be `es` for stable ECMAScript features, `esnext` for ECMAScript proposals and `web` for other web standards.
```

Comment:
I think that splitting `CONTRIBUTING.md` is not required - it's not a too big document.

Reply:
> I think that splitting `CONTRIBUTING.md` is not required - it's not a too big document.

Because there is a lot of content that could be added to this category in the future (like the currently incomplete documentation contribution guide)